exporter/fileexporter/file_exporter.go

```diff
@@ -101,3 +135,19 @@ func (e *fileExporter) Start(context.Context, component.Host) error {
 func (e *fileExporter) Shutdown(context.Context) error {
 	return e.file.Close()
 }
+
+func buildMarshaler(marshalType string) (ptrace.Marshaler, pmetric.Marshaler, plog.Marshaler) {
+	if strings.ToLower(marshalType) == marshalTypeProto ||
+		strings.ToLower(marshalType) == marshalTypeProtobuf {
+		return ptrace.NewProtoMarshaler(), pmetric.NewProtoMarshaler(), plog.NewProtoMarshaler()
+	}
+	return ptrace.NewJSONMarshaler(), pmetric.NewJSONMarshaler(), plog.NewJSONMarshaler()
+}
+
+func isJSONData(conf *Config) bool {
+	if strings.ToLower(conf.MarshalType) == marshalTypeProto ||
+		strings.ToLower(conf.MarshalType) == marshalTypeProtobuf {
+		return false
+	}
+	return true
+}
```

Comment:
If we remove one of `proto` or `protobuf`, I think we don't need this function. It should be easy enough to just check vs two possible values.